pkgs/tools/typesetting/tex/texlive/bin.nix

```
@@ -207,7 +218,16 @@ core-big = stdenv.mkDerivation { #TODO: upmendex
   # we use static libtexlua, because it's only used by a single binary
   postConfigure = let
     luajit = lib.optionalString withLuaJIT ",luajit";
-  in ''
+  in
+  lib.optionalString (stdenv.hostPlatform != stdenv.buildPlatform)
+  # without this, the native builds attempt to use the binary
+  # ${target-triple}-gcc, but we need to use the wrapper script that ensures
+  # the use of the correct libgcc.
```

```suggestion
  # the use of the correct gcc.
```

Unless you really meant `libgcc_s.so`... if you did please explain further because I don't quite follow there.

---

I guess I meant libc. without this, the build of pkgsCross.texlive.bin.core-big on x86_64-linux fails with
```
error: builder for '/nix/store/hiqrcw6218gcqpr4lj35q80xvs5czi5l-texlive-core-big.bin-aarch64-unknown-linux-gnu-2022.drv' failed with exit code 77;
       last 10 log lines:
       > checking whether make sets $(MAKE)... yes
       > checking whether make supports nested variables... yes
       > checking whether make supports the include directive... yes (GNU style)
       > checking for x86_64-unknown-linux-gnu-gcc... x86_64-unknown-linux-gnu-gcc
       > checking whether the C compiler works... no
       > configure: error: in `/build/texlive-20220321-source/WorkDir/libs/luajit/native':
       > configure: error: C compiler cannot create executables
       > See `config.log' for more details
       > configure: error: ../../../../libs/luajit/native/configure failed for native
       > /nix/store/pa9n9glr7zgf9cqr91r07h3hd6iv870h-stdenv-linux/setup: line 136: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/hiqrcw6218gcqpr4lj35q80xvs5czi5l-texlive-core-big.bin-aarch64-unknown-linux-gnu-2022.drv'.
```
and using the `x86_64-unknown-linux-gnu-gcc` in PATH manually results in
```
       > /nix/store/nydwzhllkq0a21dny69zdjczh6v275lb-binutils-2.40/bin/ld: cannot find crt1.o: No such file or directory
       > /nix/store/nydwzhllkq0a21dny69zdjczh6v275lb-binutils-2.40/bin/ld: cannot find crti.o: No such file or directory
       > collect2: error: ld returned 1 exit status
```
(crt1.o and crti.o belong to libc). Since the wrapper script does many other important things, leaving this half sentence out is probably best to avoid unnecessarily causing confusion.